ci-operator/step-registry/telcov10n/sno-hub-ztp/telcov10n-sno-hub-ztp-workflow.yaml

```diff
@@ -10,5 +10,8 @@ workflow:
     post:
     - ref: wait
     - chain: baremetal-lab-post
+    env:
+      TIMEOUT: "+30 min"
```

Comment:
Remove [this line](https://github.com/ccardenosa/release/blob/master/ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__sno-hub-ztp-4.17-latest.yaml#L54) then